aws/resource_aws_rds_cluster_endpoint_test.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSRDSClusterEndpoint_importBasic(t *testing.T) {
```

Comment:
Nit: We now prefer to skip creating "import" specific acceptance tests and instead move the `ImportState` `TestStep`s to each acceptance test (saves creating infrastructure twice in this case). 👍